platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h

```diff
@@ -339,7 +339,7 @@
 #define GPIO_UART4_TX    GPIO_UART4_TX_4    /* PD1 */
 
 #define GPIO_USART6_RX   GPIO_USART6_RX_2   /* PG9  */
-#define GPIO_USART6_TX   GPIO_USART6_TX_2   /* PG14 */
+#define GPIO_USART6_TX   (GPIO_ALT|GPIO_AF8|GPIO_SPEED_100MHz|GPIO_PUSHPULL|GPIO_PORTG|GPIO_PIN14)   /* PG14 */
```

Comment:
This is the definition of  GPIO_USART6_TX_2

Reply:
yes this was just for testing what was going on on the wire.. it is the same definition but I removed an internal pull_up that was distorting the waveform like in the first image above.